docs/Setup/ca-setup.md

```diff
@@ -0,0 +1,182 @@
+
+The _Certificate Authority_ (CA) provides a number of certificate services to users of a blockchain.  More specifically, these services relate to _user enrollment_, _transactions_ invoked on the blockchain, and _TLS_-secured connections between users or components of the blockchain.
+
+This guide builds on either the [fabric developer's setup](../dev-setup/devenv.md) or the prerequisites articulated in the [fabric network setup](Network-setup.md) guide. If you have not already set up your environment with one of those guides, please do so before continuing.
+
+### Enrollment Certificate Authority
+
+The _enrollment certificate authority_ (ECA) allows new users to register with the blockchain network and enables registered users to request an _enrollment certificate pair_.  One certificate is for data signing, one is for data encryption.  The public keys to be embedded in the certificates have to be of type ECDSA, whereby the key for data encryption is then converted by the user to be used in an [ECIES](https://en.wikipedia.org/wiki/Integrated_Encryption_Scheme) (Elliptic Curve Integrated Encryption System) fashion.
+
+### Transaction Certificate Authority
+
+Once a user is enrolled, he or she can also request _transaction certificates_ from the _transaction certificate authority_ (TCA). These certificates are to be used for deploying Chaincode and for invoking Chaincode transactions on the blockchain.  Although a single _transaction certificate_ can be used for multiple transactions, for privacy reasons it is recommended that a new _transaction certificate_ be used for each transaction.
+
+### TLS Certificate Authority
+
+In addition to _enrollment certificates_ and _transaction certificates_, users will need _TLS certificates_ to secure their communication channels.  _TLS certificates_ can be requested from the _TLS certificate authority_ (TLSCA).
+
+## Configuration
+
+All CA services are provided by a single process, which can be configured by setting parameters in the CA configuration file `membersrvc.yaml`, which is located in the same directory as the CA binary.  More specifically, the following parameters can be set:
+
+- `server.gomaxprocs`: limits the number of operating system threads used by the CA.
+- `server.rootpath`: the root path of the directory where the CA stores its state.
+- `server.cadir`: the name of the directory where the CA stores its state.
+- `server.port`: the port at which all CA services listen (multiplexing of services over the same port is provided by [GRPC](http://www.grpc.io)).
+
+Furthermore, logging levels can be enabled/disabled by adjusting the following settings:
+
+- `logging.trace` (off by default, useful for debugging the code only)
+- `logging.info`
+- `logging.warning`
+- `logging.error`
+- `logging.panic`
+
+Alternatively, these fields can be set via environment variables, which---if set---have precedence over entries in the yaml file.  The corresponding environment variables are named as follows:
```

Comment:
@srderson  this two spaces thing is not my editing/editor;-) happy to fix them, though.